src/Compilers/CSharp/Portable/CodeGen/EmitAddress.cs

```diff
@@ -95,6 +100,31 @@ private LocalDefinition EmitAddress(BoundExpression expression, AddressKind addr
             return null;
         }
 
+        private void EmitComplexConditionalReceiver(BoundComplexConditionalReceiver expression)
```

Comment:
Consider adding "Address" suffix to the function name. Otherwise functions that emit an address and emit a value are overloaded, but they are not doing similar things.